packages/ag-charts-enterprise/src/features/context-menu/contextMenu.ts

```diff
@@ -67,16 +59,14 @@ export class ContextMenu extends _ModuleSupport.BaseModuleInstance implements _M
 
     // Global shared state
     private static contextMenuDocuments: Document[] = [];
```

Comment:
It's unclear to me whether we want to keep `contextMenuDocuments` as static or change it to non-static. AFAICT we want to keep it static because this variable is used to avoid adding CSS to the HTML document multiple times.

Reply:
`static` is fine for this use-case for sure - we take same same approach for other CSS injection cases (infact we should probably factor those into one codified set of utilities) :+1: